.github/workflows/update-latest-images.yml

```diff
@@ -0,0 +1,74 @@
+name: Update base images
+
+
+on:
+  workflow_dispatch:
+    inputs:
+      params:
+        description: 'Additional parameters to updateBaseImages.sh'
+        default: ''
+        required: false
+      branch:
+        description: 'Project branch'
+        default: 'master'
+        required: true
+
+jobs:
+  build:
+    name: Check if images need to be updated to latest version
+    runs-on: ubuntu-20.04
+    steps:
+      -
+        name: "Checkout source code"
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: "Set up QEMU"
+        uses: docker/setup-qemu-action@v1
+      -
+        name: "Set up Docker Buildx"
+        uses: docker/setup-buildx-action@v1
+      -
+        name: "Docker quay.io Login"
```

Comment:
do we need docker logins for a script that just generates PRs?

Reply:
I guess not too :smiley: